### PR TITLE
PSR-330 individual buyer nino page

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/controllers/nonsipp/landorpropertydisposal/IndividualBuyerNinoNumberController.scala [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    landOrPropertyIndex: Max5000,
+    disposalIndex: Max50,
+    mode: Mode
+  ): FormPageViewModel[ConditionalYesNoPageViewModel] =
+    FormPageViewModel[ConditionalYesNoPageViewModel](
+      "individualBuyerNinoNumber.title",
+      Message("individualBuyerNinoNumber.heading", "John Doe"),
+      ConditionalYesNoPageViewModel(
+        yes = YesNoViewModel
+          .Conditional(Message("individualBuyerNinoNumber.yes.conditional", "John Doe"), FieldType.Input),
```
Comment (kberec) [inline app/controllers/nonsipp/landorpropertydisposal/IndividualBuyerNinoNumberController.scala:116]: I don't think this is right. Name should be the individual name that was typed in on the previous page.

File: app/navigation/nonsipp/LandOrPropertyDisposalNavigator.scala [hunk 2]
```diff
@@ -45,16 +45,21 @@ object LandOrPropertyDisposalNavigator extends JourneyNavigator {
...
-          controllers.routes.UnauthorisedController.onPageLoad()
         case Some(IdentityType.Individual) =>
           controllers.routes.UnauthorisedController.onPageLoad()
         case Some(IdentityType.UKCompany) =>
           controllers.routes.UnauthorisedController.onPageLoad()
         case Some(IdentityType.UKPartnership) =>
           controllers.routes.UnauthorisedController.onPageLoad()
+        case Some(IdentityType.Other) =>
+          controllers.routes.UnauthorisedController.onPageLoad()
+        case Some(IdentityType.Other) =>
```
Comment (kberec) [inline app/navigation/nonsipp/LandOrPropertyDisposalNavigator.scala:56]: This is now duplicated.

File: app/navigation/nonsipp/LandOrPropertyDisposalNavigator.scala [hunk 3]
```diff
@@ -45,16 +45,21 @@ object LandOrPropertyDisposalNavigator extends JourneyNavigator {
...
+          controllers.routes.UnauthorisedController.onPageLoad()
       }
 
+    case IndividualBuyerNinoNumberPage(srn, landOrPropertyIndex, disposalIndex) =>
```
Comment (kberec) [inline app/navigation/nonsipp/LandOrPropertyDisposalNavigator.scala:60]: Do you need to add navigation from the individual name page?

File: app/controllers/nonsipp/landorpropertydisposal/IndividualBuyerNinoNumberController.scala (hunk 1 above)
Comment (narendrapaduchuri) [reply app/controllers/nonsipp/landorpropertydisposal/IndividualBuyerNinoNumberController.scala:116]: Hi @kberec , previous page is not yet merge in to main. I will fix all in one go. Just hard coded for instance.

Comment (kberec) [reply app/controllers/nonsipp/landorpropertydisposal/IndividualBuyerNinoNumberController.scala:116]: Looks it was just merged in.

**Pull request merged**